components/dropdown/dropdown-content-mixin.js

```diff
@@ -264,6 +264,13 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 		this.opened = true;
 	}
 
+	async requestResize() {
```

Comment:
Its probably worth updating `__onResize` to call this method.

Reply:
Done